webrtc.html

```diff
@@ -1653,8 +1639,26 @@
                 to acquire those resources. The session descriptions MUST
                 remain usable by <code>setLocalDescription</code> without
                 causing an error until at least the end of the fulfillment
-                callback of the returned promise. Calling this method is needed
-                to generate the ICE username fragment and password.</p>
+                callback of the returned promise.</p>
+                <p>Creating the SDP MUST follow the appropriate process for 
+                generating an offer described in [[!JSEP]].
+                As an offer, the generated SDP will contain the full set of
+                codec/RTP/RTCP capabilities supported by the session (as 
+                opposed to an answer, which will include only a specific 
+                negotiated subset to use). In the event
+                <code>createOffer</code> is called after the session is
+                established, <code>createOffer</code> will generate an offer
+                that is compatible with the current session, incorporating any
+                changes that have been made to the session since the last
+                complete offer-answer exchange, such as addition or removal of
+                tracks. If no changes have been made, the offer will include
+                the capabilities of the current local description as well as
+                any additional capabilities that could be negotiated in an
+                updated offer.</p>
+                <p>The generated SDP will also contain the <a>ICE agent</a>s
+                <code>usernameFragment</code>, <code>password</code> and
```

Comment:
`usernameFragment` and `password` could link to the corresponding members of `RTCIceParameters`. `ice options` isn't code so I don't know if it makes sense in a code block.